CADD.pm

```diff
@@ -128,13 +129,15 @@ sub new {
     
     # using snv/indels files on SV can slow down VEP exponentially
     if ( grep( /^sv$/, @param_keys ) && ( grep( /^snv$/, @param_keys ) || grep( /^indels$/, @param_keys ) ) ) {
-      warn "WARNING: Using snv=<file> and/or indels=<file> with structural variant can increase run time exponentially.".
+      warn "WARNING: Using snv=<file> and/or indels=<file> with structural variant can increase run time exponentially. ".
```

Comment:
Suggesting we remove the space at the end of this line

Reply:
This space is actually used for separator between two lines as they are in the same paragraph. Its printed lie this - 
```
Using snv=<file> and/or indels=<file> with structural variant can increase run time exponentially. Consider creating separate input files for SNV/indels and SV and use appropriate CADD annotation file.
```